elasticsearch/src/main/java/org/hibernate/search/elasticsearch/schema/impl/DefaultElasticsearchSchemaTranslator.java

```diff
@@ -145,6 +148,12 @@ private void addPropertyMapping(ElasticsearchMappingBuilder mappingBuilder, Docu
 		mappingBuilder.setPropertyAbsolute( propertyPath, propertyMapping );
 	}
 
+	private void addStoreOption(DocumentFieldMetadata fieldMetadata, PropertyMapping propertyMapping) {
```

Comment:
What's the reason for having this in a separate method but not doing the same for boost?